docs/unique-design-challenges.md

```diff
@@ -13,6 +13,7 @@ Nostr, given its nature, presents some unique design challenges and personal rew
 3. Relays
 4. Zaps (if implemented)
 5. NIPs (and why some things are supported by some clients and not others)
+6. Deletion across tens to hundreds of relays (whereas Twitter has a sole database)
```

Comment:
I would be more generic, without mentioning only Twitter, e.g.:
_whereas closed platforms has a sole centralized database_